staged_dependencies.yaml

```diff
@@ -12,12 +12,6 @@ upstream_repos:
   insightsengineering/teal.data:
     repo: insightsengineering/teal.data
     host: https://github.com
-  insightsengineering/scda:
-    repo: insightsengineering/scda
-    host: https://github.com
-  insightsengineering/scda.2022:
-    repo: insightsengineering/scda.2022
-    host: https://github.com
```

Comment:
changing `staged_dependencies.yaml` need to be mirrored in other packages also as staged.dependencies is symmetrical.


https://github.com/insightsengineering/scda/blob/097dcfcb3bfb02e85730fbd2bc575114c26b5ac1/staged_dependencies.yaml#L15

https://github.com/insightsengineering/scda.2022/blob/13284e884111bf59be225196171da5a0551956e0/staged_dependencies.yaml#L27

Reply:
I wasn't aware of that. Thanks

Reply:
https://github.com/insightsengineering/scda.2022/pull/121
https://github.com/insightsengineering/scda/pull/123

Reply:
@gogonzo I added to PRs - if you can accept it, then we can merge all 3